ghcide/src/Development/IDE/GHC/Compat/Outputable.hs

```diff
@@ -220,17 +224,21 @@ type ErrMsg  = MsgEnvelope DecoratedSDoc
 type WarnMsg  = MsgEnvelope DecoratedSDoc
 #endif
 
+#if MIN_VERSION_ghc(9,5,0)
 mkPrintUnqualifiedDefault :: HscEnv -> GlobalRdrEnv -> PrintUnqualified
 mkPrintUnqualifiedDefault env =
-#if MIN_VERSION_ghc(9,5,0)
   mkNamePprCtx ptc (hsc_unit_env env)
     where
       ptc = initPromotionTickContext (hsc_dflags env)
 #elif MIN_VERSION_ghc(9,2,0)
+mkPrintUnqualifiedDefault :: HscEnv -> GlobalRdrEnv -> PrintUnqualified
+mkPrintUnqualifiedDefault env =
   -- GHC 9.2 version
   -- mkPrintUnqualified :: UnitEnv -> GlobalRdrEnv -> PrintUnqualified
   mkPrintUnqualified (hsc_unit_env env)
 #else
+mkPrintUnqualifiedDefault :: HscEnv -> GlobalRdrEnv -> PrintUnqualified
+mkPrintUnqualifiedDefault env =
   HscTypes.mkPrintUnqualified (hsc_dflags env)
 #endif
```

Comment:
This seems like there is no benefit at all here?

Last time I checked, the guidance on CPP in code is, minimal region possible. This change allows that the three type signatures are out of sync.

Reply:
This change was to avoid a stylish-haskell error. However, duplicating the signature was unnecessary, and I have removed that from the CPP now.